### DISCUSSION TITLE
tft 3.5 9488/9486 tuch problem

### DISCUSSION THREAD

**Discussion by shalom198** [Bodmer/TFT_eSPI#3109] [Q&A - Touch]
iam working with tft2.4 with tuch works fine
now i connected tft 3.5 9486 
the display works fine (change in the user setup.h) 
but the tuch dosnt works at all, the same xpt2046 controller for the tuch

what did i do wrong ?

**Comment by Bodmer** [comment]
It is difficult to advise on this without a lot more information on how it is wired and set up.  The Test_Touch_Controller example may help diagnose the problem.

Make sure the display actually has the tought screen fitted and the XPT2040 controller fitted.  In another controller is fitted then an alternative touch library will be needed.
